nfs-ganesha/build/build_deb

```diff
@@ -0,0 +1,85 @@
+#! /usr/bin/bash
+set -ex
+
+# Only do actual work when we are a DEB distro
+if test "$DISTRO" != "debian" -a "$DISTRO" != "ubuntu"; then
+    exit 0
+fi
+
+
+## Install any setup-time deps
+# TODO -- upload install-deps.sh script that installs the setup time deps
+# to upstream repo or update this section to install these packages
+
+# We need this for mk-build-deps
+sudo apt-get install -y equivs cmake git
+
+## Setup the pbuilder
+# TODO remove if you do not want to use pbuilders
+setup_pbuilder
+
+
+## Get some basic information about the system and the repository
+# TODO -- update this to get the proper VERSION/REVISION
+DEB_ARCH=$(dpkg-architecture -qDEB_BUILD_ARCH)
+VERSION=`grep -om 1 "[0-9]\.[0-9]-dev-[0-9]" include/config.h`
+
+## Build the source tarball
+
+git checkout $NFS_GANESHA_BRANCH
```

Comment:
you may want to do this in the yaml configuration file to get the desired branch already checked out (instead of doing it manually here)